src/app/modules/dashboard/components/course-item/course-item.component.html

```diff
@@ -1,15 +1,21 @@
-<li class="course-item generic-container">
-
+<li
+  [ngClass]="course.topRated ? 'course-item__top-rated' : ''"
+  class="course-item generic-container">
     <div class="course-item__general-info">
-      <h3 class="course-item__title">{{course.title}}</h3>
+      <h3 class="course-item__title"
+      ><span class="course-item__title-text">{{course.title | uppercase}}</span>
+      <mat-icon
+        *ngIf="course.topRated"
+        class="course-item__top-rated-icon">star</mat-icon>
+      </h3>
       <div class="course-item__details">
         <div class="agmp-menu-container with-icon course-item__duration-menu">
           <mat-icon class="agmp-menu-icon">access_time</mat-icon>
-          <span class="agmp-menu-text course-item__details-text">{{course.durationMin}}</span>
+          <span class="agmp-menu-text course-item__details-text">{{course.durationMin | timeNormalizer}}</span>
         </div>
-        <div class="agmp-menu-container with-icon course-item__creation-menu">
+        <div [courseStatus]="course.creationDate" class="agmp-menu-container with-icon course-item__creation-menu">
```

Comment:
Let's add it on component selector as we discussed earlier

Reply:
How should I get this div element if we put it on  <agmp-course-item>?